dali/pipeline/operators/transpose/cutt/CudaMemcpy.cu

```diff
@@ -0,0 +1,153 @@
+/******************************************************************************
+MIT License
+
+Copyright (c) 2016 Antti-Pekka Hynninen
+Copyright (c) 2016 Oak Ridge National Laboratory (UT-Batelle)
```

Comment:
how about our copyright notice? at least you changed the include paths

Reply:
I add ours only to the modified files
So I'll add ours to this file

Reply:
Done